test/parallel/test-fs-read-stream.js

```diff
@@ -35,7 +35,13 @@ const rangeFile = fixtures.path('x.txt');
   let paused = false;
   let bytesRead = 0;
 
-  const file = fs.createReadStream(fn);
+  const file = fs.createReadStream(fn, {
+    fs: {
+      open: common.mustCall(fs.open),
+      read: common.mustCallAtLeast(fs.read, 1),
+      close: common.mustCall(fs.close),
+    }
+  });
```

Comment:
I think we need to split this test into two. One that uses the default, and one that uses the passed in `fs` object. I'm ok in it being as simple as the write one.

Reply:
Updated